src/SFA.DAS.ApprenticeCommitments.Jobs.Functions/Infrastructure/Startup.cs

```diff
@@ -41,10 +42,16 @@ public override void Configure(IFunctionsHostBuilder builder)
                 configuration.LogDiagnostics();
 
                 configuration.AdvancedConfiguration.Conventions()
+                    .DefiningMessagesAs(t => t.Namespace?.StartsWith("SFA.DAS.Apprentice.LoginService.Messages") == true ||
+                                             t.Name == nameof(ForceAutoEventSubscription))
+
```

Comment:
Just a note, because we are defining by explicit Namespaces, any Events consumed from different packages will subsequently need changes  to this code